sidechain/consensus/slots/src/lib.rs

```diff
@@ -151,6 +151,15 @@ pub trait SimpleSlotWorker<B: ParentchainBlock> {
 	/// Remove when #447 is resolved.
 	fn allow_delayed_proposal(&self) -> bool;
 
+	/// Trigger the import of the latest parentchain block.
+	///
+	/// Returns the header of the latest imported block. In case no block was imported with this trigger,
+	/// the `current_latest_imported_header` is returned.
+	fn import_latest_parentchain_block(
+		&self,
+		current_latest_imported_header: &B::Header,
+	) -> Result<B::Header, ConsensusError>;
```

Comment:
New method on the SimpleSlotWorker trait, since we have some implementation in this trait (see below). So the implementation of this is in `Aura`.